### PR TITLE
Backup `/etc/updatedb.conf` in CSI if changes were made

### PR DESCRIPTION
**Pull request by abowloflrf**
Signed-off-by: Ruofeng Lei <ruofenglei@outlook.com>

### Ⅰ. Describe what this PR does

#2058 

Backup old `/etc/updatedb.conf` to `/etc/updatedb.conf.backup`

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/csi/updatedbconf/register.go [hunk 1]
```diff
@@ -28,6 +28,12 @@ func Register(_ manager.Manager, cfg config.Config) error {
 		glog.Info("/etc/updatedb.conf has no changes, skip updating")
 		return nil
 	}
+	glog.Info("backup old /etc/updatedb.conf to /etc/updatedb.conf.backup")
```
Comment (cheyang) [inline pkg/csi/updatedbconf/register.go:31]: I think we need check if /etc/updatedb.conf is without any changes then backup it.

Comment (abowloflrf) [reply pkg/csi/updatedbconf/register.go:31]: Line27 has already done this? 
If config file has no change then return.

Comment (cheyang) [reply pkg/csi/updatedbconf/register.go:31]: Maybe I mislead you. I think  we should backup the version before doing any modifications by fluid in /etc/updatedb.conf(the original version).

Comment (abowloflrf) [reply pkg/csi/updatedbconf/register.go:31]: Yes, the variable `content` is read from the original version of file `/etc/updatedb.conf` without any change

Comment (cheyang) [reply pkg/csi/updatedbconf/register.go:31]: > Yes, the variable `content` is read from the original version of file `/etc/updatedb.conf` without any change

Do you plan to fix it in this PR?

**Comment by xliuqq** [comment]
ok to use prefix to label the file changed or not

**Review by cheyang — APPROVED** [review]

**Review by TrafalgarZZZ — APPROVED** [review]

**Pull request merged**